R/get_climaticVariables.R

```diff
@@ -1,8 +1,26 @@
+#' WorldClim climatic variables (min temperature, max temperature, mean precipitation)
+#'
+#' This resource is published by Fick et al. (2017) "WorldClim 2: new 1-km
+#' spatial resolution climate surfaces for global land areas". This resource
+#' represents multiple climatic variables from which we will be requiring minimum
+#' temperature, maximum temperature, and mean precipitation layers. The layers are
+#' available to download for the period 2000 - 2018 on monthly basis from WorldClim.
+#' Encoded as (°C), representing the minimum and maximum temperature per output grid
+#' cell while encoded as (mm), representing the mean precipitation per output grid cell.
+#'
```

Comment:
This documents all three resources. Could you please add a itemized list that the users learn about that they can request three different resources